components/ide-service/pkg/server/ideconfig_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	ctesting "github.com/gitpod-io/gitpod/common-go/testing"
+)
+
+func TestCreateDefiniteWorkspacePod(t *testing.T) {
```

Comment:
nit: the test name reads like a copy'n'paste oversight :upside_down_face:

Reply:
🙏 Rebased to change test name @geropl